### PR TITLE
flight potions can no longer be mass produced

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
Botany can no longer farm for, and mass produce, flight potions

## Why It's Good For The Game
flight is a highly impactful game mechanic, especially given i removed the restriction between it and having a backpack awhile back. Allowing it to be easily mass-produced, and have angel wings given to the entire crew, is disruptive. This does not stop flight potion from occuring in floorpills.

## Testing Photographs and Procedure
unecessary

## Changelog
:cl:
tweak: flight potion now has can_synth set to FALSE
/:cl:

### CODE REVIEW

**Comment by ToasterBan** [comment]
did you accidentally branch from your shield gen nerf branch?

**Comment by zeskorion** [comment]
> did you accidentally branch from your shield gen nerf branch?

shhhhh

i never make mistakes

**Comment by Pirill** [comment]
But since strange seeds got practically removed from the game, flight potions are almost never seen in botany anymore. What is this PR even?

**Comment by Fronsis** [comment]
> But since strange seeds got practically removed from the game, flight potions are almost never seen in botany anymore. What is this PR even?

This, 100% haven't seen mass produced flight potion in a looong time, botanists needs to communicate with miners to get the strange seeds if they want to do it

**Comment by yurara** [comment]
Yes, theorically botanists can spam flight potions. But as people said, it will never happen due to the strange seed nurf.

**Comment by EvilDragonfiend** [comment]
Yes, theoretically botanists can spam flight potions. But as people said, it will never happen due to the strange seed nerf.

**Comment by EvilDragonfiend** [comment]
and the real spamming source for flight potion in the current state of the game is from xenobio, not from the botany.

**Comment by Mat05usz** [comment]
Pointless change that does absolutely nothing since the strange seeds removal. Now flight is one of the rarest 'prizes' when you get the mining to cooperate with botany. In reality what you're doing is limit the already severely limited interactions between botanist and mining by removing another thing from the pool, thus making RP interaction between both departments even less possible, in exchange providing literally nothing. Poor thought out removal.

Edit: 
To counter the 'arguments' in the 'Why It's Good For The Game' part:

> flight is a highly impactful game mechanic

Simply wrong, if it was everyone would be a moth. Grav gen not working happens rarely, and if it does it's usually intentional.

> especially given i removed the restriction between it and having a backpack awhile back

thank you?

> Allowing it to be easily mass-produced

it cannot be **easily** mass-produced due to reasons I stated in my reply.

>  have angel wings given to the entire crew, is disruptive

Now that's disruptive but a fucktone of dumbass viro symptoms aren't? Given how rare wings are, and how frequent viro outbreaks are, that's not really a valid argument.

And now, everyone having wings is not disruptive, it's a minor visual inconvenience at best, and a 'free' movement at no grav at worst.

> This does not stop flight potion from occuring in floorpills

amazing

So basically you have no idea how frequent something is, or what does it take to happen, but you thought about it and decided to remove?

**Comment by zeskorion** [comment]
> Pointless change that does absolutely nothing since the strange seeds removal. Now flight is one of the rarest 'prizes' when you get the mining to cooperate with botany. In reality what you're doing is limit the already severely limited interactions between botanist and mining by removing another thing from the pool, thus making RP interaction between both departments even less possible, in exchange providing literally nothing. Poor thought out removal.

There are plenty of other chemicals to desire strange seeds for. Removing this won't make it significantly less appealing

> > flight is a highly impactful game mechanic
> 
> Simply wrong, if it was everyone would be a moth. Grav gen not working happens rarely, and if it does it's usually intentional.

Moths don't get full flight. i'm talking 100% flight here


> > Allowing it to be easily mass-produced
> 
> it cannot be **easily** mass-produced due to reasons I stated in my reply.

can_synth is not only used for botany production. Science can mass produce these using an odysseus and a bit of xenobio, currently

> > have angel wings given to the entire crew, is disruptive
> 
> Now that's disruptive but a fucktone of dumbass viro symptoms aren't? Given how rare wings are, and how frequent viro outbreaks are, that's not really a valid argument.

unrelated.
(also, outbreaks are a bit more common right now as people are playing around with new content, give it some time)

> And now, everyone having wings is not disruptive, it's a minor visual inconvenience at best, and a 'free' movement at no grav at worst.

giving people tableclimbing and noslips is disruptive to normal gameplay, not to mention muh immersion

> > This does not stop flight potion from occuring in floorpills

Correct.


> So basically you have no idea how frequent something is, or what does it take to happen, but you thought about it and decided to remove?

I was not aware strange seeds were nerfed. I am glad they have been. I will not be closing this PR for reasons stated above. I should have made this FALSE when i originally sanitized CAN_SYNTH

**Comment by Mat05usz** [comment]
> > Pointless change that does absolutely nothing since the strange seeds removal. Now flight is one of the rarest 'prizes' when you get the mining to cooperate with botany. In reality what you're doing is limit the already severely limited interactions between botanist and mining by removing another thing from the pool, thus making RP interaction between both departments even less possible, in exchange providing literally nothing. Poor thought out removal.
> 
> There are plenty of other chemicals to desire strange seeds for. Removing this won't make it significantly less appealing

Like what? Are you aware of botany and what it is/isn't capable of?

> > > flight is a highly impactful game mechanic
> > 
> > 
> > Simply wrong, if it was everyone would be a moth. Grav gen not working happens rarely, and if it does it's usually intentional.
> 
> Moths don't get full flight. i'm talking 100% flight here
> 
As a person who actually plays the game I'll comment that as: XD

Explanation: It's not impactful at all. You can try and justify shit in your high castle, but when you actually play the game it's not the same.

> > > Allowing it to be easily mass-produced
> > 
> > 
> > it cannot be **easily** mass-produced due to reasons I stated in my reply.
> 
> can_synth is not only used for botany production. Science can mass produce these using an odysseus and a bit of xenobio, currently

I can't yet test if setting 'can_synth = false' means noone can get flight potion, but if so, changelog needs massive changes.
> 
> > > have angel wings given to the entire crew, is disruptive
> > 
> > 
> > Now that's disruptive but a fucktone of dumbass viro symptoms aren't? Given how rare wings are, and how frequent viro outbreaks are, that's not really a valid argument.
> 
> unrelated. (also, outbreaks are a bit more common right now as people are playing around with new content, give it some time)

Unrelated to the PR, but related to the person trying to balance things.
> 
> > And now, everyone having wings is not disruptive, it's a minor visual inconvenience at best, and a 'free' movement at no grav at worst.
> 
> giving people tableclimbing and noslips is disruptive to normal gameplay, not to mention muh immersion
> 
Do you realize how no gravity works? 
I can agree with limiting stuff based on gameplay, but you seem to just change things because

> > > This does not stop flight potion from occuring in floorpills
> 
> Correct.

Wow thank you, you gave us floorpills while restricting everything else,
> 
> > So basically you have no idea how frequent something is, or what does it take to happen, but you thought about it and decided to remove?
> 
> I was not aware strange seeds were nerfed. I am glad they have been. I will not be closing this PR for reasons stated above. I should have made this FALSE when i originally sanitized CAN_SYNTH

So you have no idea what was/wasn't changed yet you are trying to change it further.

**Comment by ToasterBan** [comment]
xenobio can mass produce flight potions much faster than botany with only a little cooperation needed, essentially the same as botany

**Comment by PowerfulBacon** [comment]
Maintainers have no strong opinions on this one, going to vote.

**Comment by williamcll** [comment]
And this is game disruptive, how? it's not that different to wing cybernetics and is only useful when gravity generator goes down. Also those wings don't work in space anyway.

**Comment by PerishedFraud** [comment]
Zesko has on occassion confessed to making PRs for the sole purpose of messing with things people like.

**Comment by EvilDragonfiend** [comment]
Your PR head only says about botany randomness while later opinion says you just had to make it not synth in R&D stuff, not in botany stuff. Your opinion isn't consistent.
if you still want to make it `can_synth=false` in case that your PR is rejected and my chem bitflag change PR is passed, you can suggest this PR again to touch it as not_synth while not touching any randomness.

**Comment by Daridine** [comment]
>I was not aware strange seeds were nerfed. I am glad they have been.  
Do not touch botany if you don't even know about one of the biggest changes they got.  

For your convenience. You get 2 strange seeds in your vendor. 
And whatever mining decides to bring back from lavaland, so, lets say 0-3.
